docs/proposals/form-metadata.md

```diff
@@ -0,0 +1,127 @@
+# Add UI metadata to catalog requests
+
+## Introduction
+Support better provision user experience for APBs by giving additional metadata about APB parameters.
+
+## Problem Description
+Currently, APB parameters are shown in random order and without any kind of grouping.  the OpenServiceBroker API utilizes json-schema data format and does not include any UI metadata.  As such, the UI cannot customize the experience and users are confronted with a bad experience which gets worse as the number of parameters increases. 
+
+## Implementation Details
+
+### APB changes
+APBs will allow an author to implement parameter order, grouping, and form field type.
+* Ordering will honor the order of parameters used in the apb.yml
+* Grouping will require the addition of a parameter field `display_group`
+* UI field types will require the addition of a parameter field `display_type`.
+
+```yaml
+name: hello-world-db-apb
+# ...snip...
+plans:
+  - name: default
+    # ...snip...
+    parameters:                           # respects order of parameters
+      - name: postgresql_database         # first parameter on form
+        title: PostgreSQL Database Name
+        type: string
+        required: True
+        default: admin
+      - name: postgresql_user             # second parameter
+        title: PostgreSQL User
+        type: string
+        required: True
+        default: admin
+        display_group: User Information   # same group as password       
```

Comment:
What's the default value of display_group? Can display group be an arbitrary string or are there categories like User Information, Bind Information, Database Information, ect...?